.github/workflows/test.yaml

```diff
@@ -72,6 +91,7 @@ jobs:
         run: |
           eval "$(conda shell.bash hook)"
           conda activate test-environment
+          python -m pip install -e .
```

Comment:
It's unfortunate to have to add this command but I guess it's unavoidable! I think the exact command run by pyctdev is `pip install --no-deps --no-build-isolation -e .` so we may want to add these two flags too, mostly for the sake of being consistent.

(I'm actually not sure that installing in editable mode is required, this is a practical default of pyctdev when running it locally to install a dev env because this is what you want but in a CI context it certainly doesn't make that much sense, if any).

Reply:
It is required to do an editable install in this case as somewhere in the tests there is an assumption that hvplot is pip installed.

Reply:
I meant I think it'd be possible to pip install it without the editable flag, but it's fine as it is since it's anyway what `pyctdev` does.